### ISSUE TITLE
Send usage stats to the cloud

### ISSUE THREAD

**Issue by DamnClin** [jhipster/jhipster-lite#4566]
Discussing with @murdos we though that it might be a good idea to send usage statistics to the cloud. That mean adding a dedicated handler to send usage to the deployed version (and a property to disable that if needed).

@jdubois where are we on stats persistence on the deployed instance?

**Comment by jdubois** [comment]
We're nowhere because I didn't work on it. What you did should be very easy to implement on my side.

Then I'm wondering now if we shouldn't have some dedicated service that can do reporting and graphs... For JHipster we still have https://yeoman.io/misc/insight working (nobody remembers this but me, probably!!), and this gives super nice charts and dashboards. Because the issue is not just sending the data, but also how can we analyse it.

**Comment by DamnClin** [comment]
I have some issue created to build dashboards from the stats we get :). I think this can be in the JHlite application since it's a cool feature for custom jhlite instances (the ones created from the `custom-jhlite` module and were you can build your own modules)

**Comment by jdubois** [comment]
Yes it's my fault, I still need to create the MongoDB instance to get the stats.

**Comment by pascalgrimaud** [comment]
@jdubois : what's the cheaper? the MongoDB instance or a SQL instance (like PostgreSQL or MySQL)

**Comment by jdubois** [comment]
I wanted to use the MongoDB API on top of Azure CosmosDB. There is a free tier that should be enough for us, so my goal is that this would be free :-)
